exercises/01_trim_right/main_test.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import "testing"
+
+func TestTrimRight(t *testing.T) {
+	for _, tc := range []struct {
+		name  string
+		input string
+		want  string
+	}{
+		{"empty", "", ""},
+		{"nothing to trim", "abc", "abc"},
```

Comment:
I also recommend to add a testing like `"a "`. Maybe a single space as well e.g. `" "`.